No module named kastore when following Development guide

Following the [Development](https://tskit.dev/msprime/docs/latest/development.html#) guide (which tells to install requirements from `requirements/development.txt`), `pytest` is falling with one error :
```shell
ImportError while importing test module '/workspaces/msprime/tests/test_dict_encoding.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/local/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_dict_encoding.py:5: in <module>
    import lwt_interface.dict_encoding_testlib
lwt_interface/dict_encoding_testlib.py:31: in <module>
    import kastore
E   ModuleNotFoundError: No module named 'kastore'
```

Thank you for your help !

--- a/requirements/development.txt
+++ b/requirements/development.txt
@@ -17,6 +17,7 @@ pytest
 pytest-cov
 pytest-xdist
 tskit>=0.5.2
+kastore
 sphinx-book-theme
 stdpopsim==0.1.2 # Make sure we have correct version of OOA model
 scipy

